CHANGELOG.md

```diff
@@ -2,9 +2,14 @@
 
 ## **[Unreleased]**
 
+## 0.10.0 - 2019-11-11
+
+Special thanks to @newpavlov and @Maxgy for their contributions!
```

Comment:
Somehow Github doesn't link automatically when people are tagged. So this will be better!

```suggestion
Special thanks to [@newpavlov](https://github.com/newpavlov) and [@Maxgy](https://github.com/Maxgy) for their contributions!
```